### ISSUE TITLE
0.9.11 throws Cannot access 'image': it is public in '<anonymous>' with gradle kotlin script

### ISSUE THREAD

**Issue by MichaelKunze** [GoogleContainerTools/jib#1110]
**Description of the issue**:
Version 0.9.11 of jib does not seem to work with Gradle Kotlin Script and throws `Cannot access 'image': it is public in '<anonymous>'`

**Expected behavior**:´
`./gradlew help` should work like it does with `0.9.10`

**Steps to reproduce**: 
Bootstrap an empty project with gradle 4.10.2 wrapper. Add a `build.gradle.kts` with the content below and execute `./gradlew help`

**Environment**: 
Gradle Warpper: gradle-4.10.2-bin
Java: openjdk version "10.0.2" 2018-07-17

**`build.gradle.kts` Configuration**:
```groovy
plugins {
    id("java")
    id("com.google.cloud.tools.jib") version "0.9.11"
}

jib {
    from {
        image = "openjdk:11-slim"
    }
    to {
        image = "0815.dkr.ecr.eu-central-1.amazonaws.com/test-repo:latest"
    }
}

```

**Log output**: 
```
* What went wrong:
Script compilation errors:

  Line 08:         image = "openjdk:11-slim"
                   ^ Cannot access 'image': it is public in '<anonymous>'

  Line 11:         image = "0815.dkr.ecr.eu-central-1.amazonaws.com/test-repo:latest"
                   ^ Cannot access 'image': it is public in '<anonymous>'

2 errors
```

**Additional Information**:

**Comment by coollog** [comment]
Hi @MichaelKunze , this was actually reported in #1060. We will be releasing a version `0.9.12` soon with the fix for this issue.

**Comment by coollog** [comment]
We have just released `v0.9.12` with the fix for this issue.
